sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -71,7 +71,7 @@ Netty and include OkHTTP client in your pom.xml.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.1.0-beta.1</version>
+  <version>1.0.0</version>
```

Comment:
Should I revert this?

Reply:
@JimSuplizio

Reply:
I'd revert this file and let whomever owns azure-ai-textanalytics that this needs to be updated before they ship. Also, if we're shipping okhttp (I honestly don't know) then this version will change.